web/client/components/geostory/layouts/Cascade.jsx

```diff
@@ -126,43 +127,50 @@ const Cascade = ({
             const sizeClassName = containerSize ? ` ms-${containerSize}` : '';
             const isMediaExpandable = containerSize === 'sm';
             const storyTheme = theme && isObject(theme) && theme || {};
-            return (<div
-                id="ms-sections-container"
-                className={`ms-sections-container${sizeClassName}`}
-                style={{
-                    ...storyTheme?.general,
-                    ...isContentFocused && { overflow: 'hidden' }
-                }}>
-                {
-                    sections.map(({ contents = [], id: sectionId, type: sectionType, cover }) => {
-                        return (
-                            <Section
-                                focusedContent={focusedContent}
-                                onVisibilityChange={onVisibilityChange}
-                                add={add}
-                                editMedia={editMedia}
-                                expandableMedia={isMediaExpandable}
-                                editWebPage={editWebPage}
-                                updateCurrentPage={updateCurrentPage}
-                                update={update}
-                                remove={remove}
-                                key={sectionId}
-                                id={sectionId}
-                                viewHeight={height}
-                                viewWidth={width}
-                                type={sectionType}
-                                mode={mode}
-                                contents={contents}
-                                cover={cover}
-                                storyTheme={storyTheme}
-                                mediaViewer={mediaViewer}
-                                contentToolbar={contentToolbar}
-                                storyFonts={storyFonts}
-                            />
-                        );
-                    })
-                }
-            </div>);
+            return (
+                <Style>
+                    {`
```

Comment:
I would avoid to add a new library for a single CSSinJS style.

- [ ] Could we use directly the `style` tag with `dangerouslySetInnerHTML`?

Something like this:

```js
<div
    id="ms-sections-container"
    className={`ms-sections-container${sizeClassName} ms-sections-hyperlinks`}
    style={{
        ...storyTheme?.general,
        ...isContentFocused && { overflow: 'hidden' }
    }}>
    {storyTheme?.link?.color && <style dangerouslySetInnerHTML={{__html: `
        .ms-sections-hyperlinks .ms-text-editor-main a,
        .ms-sections-hyperlinks .ms-text-wrapper a,
        .ms-sections-hyperlinks .ms-text-editor-main a:link,
        .ms-sections-hyperlinks .ms-text-wrapper a:link,
        .ms-sections-hyperlinks .ms-text-editor-main a:visited,
        .ms-sections-hyperlinks .ms-text-wrapper a:visited,
        .ms-sections-hyperlinks .ms-text-editor-main a:active,
        .ms-sections-hyperlinks .ms-text-wrapper a:active {
            color: ${storyTheme.link.color};
            opacity: 1;
        }
        .ms-sections-hyperlinks .ms-text-editor-main a:hover,
        .ms-sections-hyperlinks .ms-text-wrapper a:hover {
            color: ${storyTheme.link.color};
            opacity: 0.75;
        }
    `}} />}
```

- [ ] The CSS should be more specific on the selector part (see snippet above) and target only text contents. Currently it's applied also on the text inline toolbar. We should also add all related pseudo classes and we could use opacity for an hover effect.

![image](https://user-images.githubusercontent.com/19175505/92916756-17038100-f42e-11ea-9c20-468d9ed78293.png)